x-pack/plugins/features/server/feature_schema.ts

```diff
@@ -52,6 +53,10 @@ const kibanaPrivilegeSchema = Joi.object({
     all: alertingSchema,
     read: alertingSchema,
   }),
+  cases: Joi.object({
+    all: casesSchema,
+    read: casesSchema,
+  }),
```

Comment:
Can you add unit tests (e.g as done for `alerting`) in `x-pack/plugins/features/server/feature_registry.test.ts`?

Reply:
Done: [`f7a816b` (#95058)](https://github.com/elastic/kibana/pull/95058/commits/f7a816bee8479199202f61c60a86598639e2ab5f)